albumentations/augmentations/geometric/transforms.py

```diff
@@ -2038,3 +2039,146 @@ def get_params(self) -> dict[str, D4Type]:
 
     def get_transform_init_args_names(self) -> tuple[()]:
         return ()
+
+class GridElasticDeform(DualTransform):
+    """ Grid-based Elastic deformation Albumentation implementation
+
+    This class applies elastic transformations on a grid-based approach,
+    where the granularity of the distortions can be controlled using the
+    width and height of the overlaying distortion grid. Larger grid sizes
+    result in finer, less severe distortions.
+
+    Params:
+        n_grid_width (int): Number of grid cells along the width
+        n_grid_height (int): Number of grid cells along the height
+        magnitude (int): Magnitude of distortion
+        p (float): Probability of applying the transform
+        always_apply (bool | None): If true, the transform will always be applied.
+
+    Targets:
+        image, mask
+
+    Image types:
+        uint8
+    """
+
+    def __init__(
+        self,
+        n_grid_width: int,
```

Comment:
`n_grid_width` and `n_grid_height`, should be merged into one parameter `num_grid_size` or `num_grid` or `num_grid_xy` or similar.